### PR TITLE
Fix embarrassing typos

### PR DESCRIPTION
**Pull request by dandv**
Many of these typos were consistently recurring (e.g. "dosnt"). It would really help to install a spell checker in the IDE(s) used to work on the project :)

### CODE REVIEW

**Comment by hasezoey** [comment]
from what i know not placing `'` in dev-use is not a problem

**Comment by dandv** [comment]
Poor spelling, capitalization and grammar lower credibility.

File: src/internal/utils.ts [hunk 1]
```diff
@@ -133,7 +133,7 @@ export function initProperty(name: string, key: string, whatis: WhatIsIt) {
       schemas.get(name)[key] = {};
       break;
     default:
-      throw new TypeError('"whatis" is not supplied OR dosnt have a case yet!');
+      throw new TypeError('"whatis" is not supplied OR doesn't have a case yet!');
```
Comment (hasezoey) [inline src/internal/utils.ts:136]: sorry, but adding this `'` here breaks the string

```suggestion
      throw new TypeError('"whatis" is not supplied OR doesn\'t have a case yet!');
```

Comment (dandv) [reply src/internal/utils.ts]: Good catch! I made a lot of manual fixes, but also some search&replaces, and didn't notice that.

**Pull request merged**